.github/workflows/typescript-compilation.yml

```diff
@@ -0,0 +1,44 @@
+name: Typescript compilation
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'web/html/src/**'
+  pull_request:
+    paths:
+      - 'web/html/src/**'
+
+jobs:
+  typescript_compilation:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
```

Comment:
Not sure if it brings many advantages, but It seems there is a v2 available that could bring some performance benefits by caching node versions especially with self-hosted runners (IIUC): https://github.com/actions/setup-node#v2

Reply:
Upgraded to use v2, thanks a lot for the suggestion!